CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+* A new feature flag has been introduced as an environment variable to manage redaction support. If the GP2GP_REDACTIONS_ENABLED 
```

Comment:
I'd say let's keep this feature quiet for now. End users won't care too much about which message type is being sent out, and the adaptor doesn't actually send out the redaction informatio yet. This could just cause more questions.